sos/report/plugins/aap_gateway.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2024 Lucas Benedito <lbenedit@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class AAPGatewayPlugin(Plugin, RedHatPlugin):
+    short_desc = 'AAP Gateway plugin'
+    plugin_name = 'aap_gateway'
+    profiles = ('sysmgmt', 'ansible',)
+
+    packages = ('automation-gateway',
+                'automation-gateway-config')
+    commands = ('aap-gateway-manage',)
+    services = ('automation-gateway',)
+
+    def setup(self):
+        self.add_copy_spec([
+            "/var/log/nginx/access.log*",
+            "/var/log/nginx/error.log*",
```

Comment:
The `nginx` stuff is already being collected by the `nginx` plugin, if that would be triggered as well, then you could have 2 plugins collect the same thing

Reply:
Indeed, we should avoid that duplicate collection. Since `ngix` plugin collects just non-logrotated files (https://github.com/sosreport/sos/blob/main/sos/report/plugins/nginx.py#L33-L34), I think the best way is to enhance `ngix` plugin to collect `access.log*` and `error.log*`?

Reply:
The configuration made here will collect rotated logs. I am not sure if for other teams these logs are as relevant as they are for us, being the reason I did not suggest the change on the main nginx plugin.

Reply:
I don't think, this is about teams or anything like that. This could be useful for other people as well, and all the `nginx` stuff should go into the `nginx` plugin. If we have an environment that is not `aap_gateway` but the system has `nginx`, then it would be useful for that scenario. So +1 on what @pmoravec said above

Reply:
Thanks! I think `nginx` plugin will also much rather benefit from the change (of additionaly collecting logrotated logs) than harm from it (more data on slower sosreports done).